bdsbilling/src/main/java/com/appcoins/wallet/bdsbilling/repository/entity/DetailsResponseBody.kt

```diff
@@ -11,4 +11,9 @@ data class DetailsResponseBody(val items: List<ResponseProduct>) {
   data class Currency(val code: String, val symbol: String)
 
   data class PackageResponse(val name: String)
+}
+
+fun DetailsResponseBody.merge(other: DetailsResponseBody) : DetailsResponseBody {
```

Comment:
Please run the code style on this file.
Line 16 : there is an extra white space after ")"